src/components/InputBox/index.js

```diff
@@ -1,12 +1,14 @@
+/* eslint-disable react/jsx-no-duplicate-props */
+/* eslint-disable prettier/prettier */
 import React from 'react';
 
 import './Inputbox.scss';
```

Comment:
This should be in Pascal casing to match the casing of the component

Reply:
I have fixed this